### PR TITLE
Add content type to messagesets

### PR DESCRIPTION
**Pull request by gsvr**

### CODE REVIEW

**Comment by gsvr** [comment]
@imsickofmaps Please review.

File: contentstore/models.py [hunk 1]
```diff
@@ -50,6 +50,11 @@ class MessageSet(models.Model):
         Details about a set of messages that a recipient can be sent on
         a particular schedule
     """
+    CONTENT_TYPES = (
+        ("sms", 'SMS'),
```
Comment (imsickofmaps) [inline contentstore/models.py]: can we make these `text` and `audio`?

File: contentstore/models.py [hunk 2]
```diff
@@ -58,6 +63,7 @@ class MessageSet(models.Model):
     default_schedule = models.ForeignKey(Schedule,
                                          related_name='message_sets',
                                          null=False)
+    content_type = models.CharField(choices=CONTENT_TYPES, max_length=20)
```
Comment (imsickofmaps) [inline contentstore/models.py]: missing default `text` please

**Comment by imsickofmaps** [comment]
No tests?

File: contentstore/models.py (hunk 2 above)
Comment (gsvr) [inline contentstore/models.py]: wondered about that

**Comment by imsickofmaps** [comment]
:+1:

**Pull request merged**